source/assets/js/playground/split-view.ts

```diff
@@ -0,0 +1,17 @@
+import {SplitView} from '@spectrum-web-components/split-view';
+import '@spectrum-web-components/split-view/sp-split-view.js';
+
+/**
+ * Make split views responsive and stack vertically on narrow screens.
+ */
+export default function setupSplitView(): void {
```

Comment:
Nit:

```suggestion
export default function setUpSplitView(): void {
```

"setup" is the noun form, "set up" is the verb :slightly_smiling_face:

Reply:
Updated!